### DISCUSSION TITLE
主应用和子应用都是vue，想把主应用组件给子应用使用，数据可以更新，渲染为什么无法更新

### DISCUSSION THREAD

**Discussion by hex11o** [umijs/qiankun#2431] [Q&A]
主应用简单的组件代码如下：
```
<template>
  <div>
    <span>
      num = {{ num }}
    </span>
    <button @click="add">+1</button>
  </div>
</template>

<script setup lang="ts">
import { ref } from 'vue'
const num = ref(0)
const add = () => {
  num.value++
  console.log(num.value); // 打印数据有更新
}
</script>
```

通过window或者props导出在子应用使用，点击时发现数据有更新打印，但是视图上未更新，想问问是什么原因以及有解决办法么，谢谢；
<img width="674" alt="截屏2023-03-13 00 20 06" src="https://user-images.githubusercontent.com/10339244/224557854-9c9bed52-d461-4d3e-9da7-d0f303a2a0ce.png">

**Comment by battleingame** [comment]
vue实例不一致导致没有手机到依赖
